src/site/facilities/health_service.drupal.liquid

```diff
@@ -26,11 +26,11 @@
                     <ul class="usa-unstyled-list">
                         {% for location in healthService.fieldHealthServicesLocalInfo %}
                             {% assign facility = location.entity.fieldLocalFacilityOrClinic.entity %}
-                            <li class="vads-u-margin-bottom--2"><a href="/drupal{{ entityUrl.path }}/{{ facility | locationUrlConvention }}">{{ facility.fieldNicknameForThisFacility }}</a></li>
+                            <li class="vads-u-margin-bottom--2"><a href="{{ entityUrl.path }}/{{ facility | locationUrlConvention }}">{{ facility.fieldNicknameForThisFacility }}</a></li>
```

Comment:
We shouldn't have to add `/drupal` into our templates, because the `rewrite-drupal-pages` Metalsmith plugin should automatically apply that.